BUILD.md

```diff
@@ -345,19 +345,9 @@ Setup Homebrew and components
 
       brew install cmake python python3 git
 
-### Clone the Repository
+### Get the MoltenVK Library
```

Comment:
Is the MoltenVK library even required at all for building Vulkan-Loader?    (I don't think so).  You give instructions at the (new) line 350 to clone and build it, but there is no direction about how to make the Vulkan-Loader __use__ it, similar to how the GLSLANG_REPO_ROOT works.

Could you please verify that this isn't needed and I'm not missing something.

Reply:
After getting to the bottom part of the doc, I think that the MVK lib is only needed if someone wants to run the test case.  IOW, if all they want to do is build the loader, then they don't need MVK.

So I would move lines 348-350 down to just above the section that talks about exporting VK_ICD_FILENAMES.